src/search/css/SearchForm.scss

```diff
@@ -1,3 +1,27 @@
-form {
-  background: green;
+@import "inc/vars";
+
+.search-form {
+  display: flex;
+
+  label {
+    font-size: $font-size-m;
+  }
+
+  input {
+    box-sizing: border-box !important;
```

Comment:
Is is due to normalize.css which adds content-box to `input[type=search]`. I've removed the !important though.